env_test.go

```diff
@@ -1,18 +1,18 @@
 package main_test
 
 import (
-    "testing"
-    "github.com/ingydotnet/go-env"
+	"github.com/helionce/go-env"
+	"testing"
 )
 
 func TestCount(t *testing.T) {
-    if main.Count() != 1 {
-        t.Error("Expected 1")
-    }
-    if main.Count() != 2 {
-        t.Error("Expected 2")
-    }
-    if main.Count() != 3 {
-        t.Error("Expected 3")
-    }
+	if main.Count() != 1 {
```

Comment:
It's not a requirement to do so, but generally we use these test packages `github.com/stretchr/testify/assert` and `github.com/stretchr/testify/require`. Then you could do `assert.Equal(t, 1, main.Count())`.